admin_manual/configuration_ldap/ldap_proxy_cache_server_setup.rst

```diff
@@ -75,36 +78,138 @@ With that done, restart OpenLDAP by running the following command::
 
-  tail -f /var/log/syslog | grep QUERY
+-h = host adress (Example: localhost or 192.168.1.1)
+-H = host adress (Example: ldap:// or ldaps:// hostname or ip and port :389 or :636 )
+-x = simple authentication
+-b = Search Base, (Example: "cn=Users,dc=example,dc=com")
+-D = User with permissions (Example: "cn=Admin,dc=example,dc=com")
+-LLL = Show only results, no extra information
+-w = Password ("Password")
+-W = Password, will ask for password and hide your input
+"(cn=Administrator)" = Filter the search
+name = Show only this attributes
 
 If you see results including: ``"Query cachable"`` and ``"Query answered (x) times"``, then the setup works. 
 
+
+6. Configure ownCloud LDAP App
+--------------------------------
+
+Login as ownCloud admin in your ownCloud server.
+
+Click on the dropdown menu in the top-left corner next to "Files", then on Apps. 
+
+Click on "Not enabled", and enable the "LDAP user and group backend" App.
+
+Go to the admin dropdown menu in the top-right corner, select "Admin".
+
+In the administration section, click on the left side on "LDAP".
+
+
+Configure Server-Tab
+
+First enter the server address, either IP or DN.
+
+You can click on the button to detect your servers port or enter it manualy.
+
+Next, enter the dn of the user, you want to log in with and in the next line enter the password.
+
+Then you can click on "detect base dn" or enter it manualy. then click on "test base dn".
+
+If you fullfill all requirement you should get a green light and "configuration ok" message.
+
+
+Configure Users
+
+Select the objectclass for the users, for example "user".
+
+Verify your settings, you will see the number of users being found.
+
+
+Configure Login Attributes
+
+A configuration appears by default, adjusted to the users configuration, 
+if required, adjust the login paramethers additional login attributes.
+
+
+You can check users with any of the allowed login options. You can adjust them or let them the way they are.
+
+
+Configure Groups
+
+Select all the objectclasses for your groups, for example "group". Verify your settings.
+
+
+Configure Advanced
+
+Configuration Active shoud be selected.
+
+Adjust the Cache TTL (time to live) value as required.   
+
+Owncloud usualy autoselects the best settings for each AD configuration.
+
+Check if the Group-Member association is "Member (AD)".
+That's important for the users being shown in their respective groups.
+
+Select "Nested groups", if you have them.
+
+
+Configure Expert
+
+"Internal Username Attribute" 
+
+Here we need to set "cn" for the users being shown with their unique name. 
+If you leve that field empty, each user will get a unique uid as a string of numbers and letters.
+
+Clear the Username and Groupname Mapping and test your configuration by clicking on the buttons below.
+
+
+Go to admin, Users and check if all your users are listed properly and shown in the right groups.
```

Comment:
Here, I'd suggest: "Navigate to `Admin -> Users` and check if all your users are listed properly, and shown in the right groups."